src/libImaging/Jpeg.h

```diff
@@ -83,6 +83,10 @@ typedef struct {
     /* Chroma Subsampling (-1=default, 0=none, 1=medium, 2=high) */
     int subsampling;
 
+    /* Restart marker interval, in MCU blocks or MCU rows, or 0 for none */
+    unsigned int restart_marker_blocks;
+    int restart_marker_rows;
```

Comment:
Why different types?

Reply:
Those are the types libjpeg uses.  Looking through the libjpeg source code, there doesn't seem to be a good reason for it, and I agree it seems pretty nonsensical.  I can switch over to unsigned if that's preferred.

Reply:
Done.